### DISCUSSION TITLE
send_private_msg  发送长度限制

### DISCUSSION THREAD

**Discussion by xiaoshengbaiyi** [Mrs4s/go-cqhttp#991] [Q&A]
api /send_private_msg 好像存在长度限制
![1628139672623](https://user-images.githubusercontent.com/44537257/128295692-84ead20a-046f-4aa0-80bc-f96285c48dde.png)
![1628139697352](https://user-images.githubusercontent.com/44537257/128295709-1a4157db-4baa-4ce1-925d-520186093e3f.png)

**Comment by SamDudu** [comment]
我也遇到相同的问题，汉字760多个就截断了。

**Comment by xiguasamall** [comment]
我是1666个汉字就被拦截了
